providers/hubspot/webhook.go

```diff
@@ -40,6 +40,24 @@ func (c *Connector) GetRecordFromWebhookMessage(
 	return c.GetRecord(ctx, objectName, recordId)
 }
 
+func (c *Connector) ExtractEventTypeFromWebhookMessage(msg *WebhookMessage) (common.WebhookEventType, error) {
+	parts := strings.Split(msg.SubscriptionType, ".")
+	if !getRecordSupportedObjectsSet.Has(parts[0]) {
```

Comment:
Please remove this check from all the Extract functions in this file, just because we cannot get a record for that particular object, we should not error out. We can just leave the `result` blank when we send the webhook to the builder. We want people to be able to use Subscribe Actions for ALL Hubspot event types and not unnecessarily limit them.